Libs/MRML/Widgets/qMRMLThreeDViewControllerWidget.cxx

```diff
@@ -334,12 +334,12 @@ void qMRMLThreeDViewControllerWidget::updateWidgetFromMRML()
     }
 
   QStringList axesLabels;
-  axesLabels <<  d->ViewNode->GetAxisLabel(0);
   axesLabels <<  d->ViewNode->GetAxisLabel(1);
```

Comment:
Add a comment here that: in the axes widget the order of labels is: +X, -X, +Z, -Z, +Y, -Y and in the view node axis labels order is -X, +X, -Y, +Y, -Z, +Z. And maybe after each line add a comment which label is that, for example:
axesLabels <<  d->ViewNode->GetAxisLabel(1); // +X